ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -450,24 +450,23 @@ class TabLocationView: UIView {
   }
 
   private func updateURLBarWithText() {
-    guard let url = url else {
-      urlDisplayLabel.text = ""
-      return
-    }
-
-    if let internalURL = InternalURL(url), internalURL.isBasicAuthURL {
+    if let url = url, let internalURL = InternalURL(url), internalURL.isBasicAuthURL {
       urlDisplayLabel.text = Strings.PageSecurityView.signIntoWebsiteURLBarTitle
     } else {
       // Matches LocationBarModelImpl::GetFormattedURL in Chromium (except for omitHTTP)
       // components/omnibox/browser/location_bar_model_impl.cc
       // TODO: Export omnibox related APIs and use directly
-      urlDisplayLabel.text = URLFormatter.formatURL(
-        url.scheme == "blob" ? URLOrigin(url: url).url?.absoluteString ?? "" : url.absoluteString,
-        formatTypes: [
-          .trimAfterHost, .omitHTTP, .omitHTTPS, .omitTrivialSubdomains, .omitDefaults,
-        ],
-        unescapeOptions: .normal
-      )
+      if let url = url {
+        urlDisplayLabel.text = URLFormatter.formatURL(
+          url.scheme == "blob" ? URLOrigin(url: url).url?.absoluteString ?? "" : url.absoluteString,
+          formatTypes: [
+            .trimAfterHost, .omitHTTP, .omitHTTPS, .omitTrivialSubdomains, .omitDefaults,
+          ],
+          unescapeOptions: .normal
+        )
+      } else {
+        urlDisplayLabel.text = ""
```

Comment:
Do we need this part? It wasn't part of the original function

Reply:
The original function does: `URLFormatter.formatURL(url?.absoluteString ?? "", ....)` which will return empty string if passed an empty string.

So instead of calling the function with an empty string, I just assigned it here.